### ISSUE TITLE
Add tags to challenges 161-180

### ISSUE THREAD

**Issue by kevmoens** [kevmoens/Ellabit#13]
On each challenge there is a property named Tags.

```
        private Dictionary<string, string> _tags = new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase) { {"Math","Add"}, { "Level", "1" } };
        public Dictionary<string, string> Tags { get => _tags; set => _tags = value; }
````

Tags describe the type of coding being done in each challenge.
In the code above is describing the challenge uses Math to Add and Level is 1.  
You can add as many tags as you like to help describe the challenge.
Add a Level tag to each challenge. 
Currently levels are described as:
Level 1 = beginner concept
Level 2 = Use of beginner concept with more detailed knowledge to code
Level 3 = Something that requires in depth knowledge to code
Level 4 = This is a complex challenge that will take more than 10 minutes to code for an advanced developer
Level 5 = This is a senior developer that will take more than 15-20 minutes to code

**Comment by Samkituka** [comment]
### Hello,
I am interested in your project.
What kind of tags should  add to challenges 161-180?

**Comment by kevmoens** [comment]
I added details into the issue.
Let me know if you need any more details.
Thanks for helping.
    On Thursday, October 13, 2022 at 11:17:36 AM CDT, Samkituka ***@***.***> wrote:  
 
 


Hello,

I am interested in your project.
What kind of tags should add to challenges 161-180?

**Comment by Samkituka** [comment]
Thanks for the clarification.

**Comment by kevmoens** [comment]
When submitting code, please make sure your code compiles before adding a pull request.

**Comment by kevmoens** [comment]
@Samkituka 
I went through the last 5 challenges and most of them I needed to change the tags.  I suggest you go write the code to solve the challenge and create tags that align with the code to solve the challenge.

**Comment by kevmoens** [comment]
fixed by #26
